### PR TITLE
Sergkanz/context tags cleanup

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
The idea of the schema cleanup is to only expose fields available in Application Analytics and making sense for Application Insights for services.

Thus marking those fields obolete

### CODE REVIEW

File: src/Core/Managed/Shared/Extensibility/Implementation/DeviceContext.cs [hunk 1]
```diff
@@ -64,28 +67,31 @@ public string Model
         /// Gets or sets the <a href="http://www.iana.org/assignments/ianaiftype-mib/ianaiftype-mib">IANA interface type</a> 
         /// for the internet connected network adapter.
         /// </summary>
+        [Obsolete("Use custom properties.")]
         public string NetworkType
         {
-            get { return this.tags.GetTagValueOrNull(ContextTagKeys.Keys.DeviceNetwork); }
-            set { this.tags.SetTagValueOrRemove(ContextTagKeys.Keys.DeviceNetwork, value); }
+            get { return this.properties.GetTagValueOrNull("ai.device.network"); }
```
Comment (dnduffy) [inline src/Core/Managed/Shared/Extensibility/Implementation/DeviceContext.cs]: Why get rid of the "constants" for the keys?

**Comment by dnduffy** [comment]
:shipit:

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 This changelog will be used to generate documentation on [release notes page](http://azure.microsoft.com/en-us/documentation/articles/app-insights-release-notes-dotnet/).
 
+- Context properties `NetworkType`, `ScreenResolution` and `Language` marked as obsolete. Please use custom properties to report network type, screen resolution and language. Values stored in these properties will be send as custom properties.
```
Comment (abaranch) [inline CHANGELOG.md]: >- Context [](start = 0, length = 9)

This should go inside 2.2-beta2

**Comment by abaranch** [comment]
:shipit:

File: CHANGELOG.md (hunk 2 above)
Comment (SergeyKanzhelev) [inline CHANGELOG.md]: Sure, I'll fix it

File: src/Core/Managed/Shared/Extensibility/Implementation/DeviceContext.cs (hunk 1 above)
Comment (SergeyKanzhelev) [inline src/Core/Managed/Shared/Extensibility/Implementation/DeviceContext.cs]: they are obsolete thus - not generated in the list of constants

**Pull request merged**